styled.css

```diff
@@ -0,0 +1,308 @@
+@font-face {
+  font-family: NexaBlack;
+  src: url("./fonts/NexaBlack.otf");
+}
+@font-face {
+  font-family: NexaBold;
+  src: url("./fonts/NexaBold.otf");
+}
+@font-face {
+  font-family: NexaLight;
+  src: url("./fonts/NexaLight.otf");
+}
+@font-face {
+  font-family: Nexa;
+  src: url("./fonts/NexaRegular.otf");
+}
+
+body {
+  background-color: #0e1016;
+  font-family: Nexa;
+  color: #fff;
+}
+.container {
+  max-width: 1170px;
+  width: 100%;
+  margin: 0 auto;
+}
+
+.innerWrapper {
+  display: flex;
+  justify-content: center;
+}
+.imageDesign {
+  max-width: 342px;
+  width: 100%;
+}
+.description {
+  color: #fff;
+  font-size: 16px;
+}
+
+.button {
+  background-color: #0d4df2;
+  border: 1.5px solid #0d4df2;
+  height: 16px;
+  font-size: 14px;
+  padding: 15px 30px;
+  color: #fff;
+  border-radius: 8px;
+  width: 20%;
+  margin: 1px;
+  width: 20%;
+}
+
+.h3Custom {
+  font-family: NexaBold;
+  font-size: 44px;
+  font-weight: 700;
+  line-height: 66px;
+  letter-spacing: normal;
+  color: #c6cddd;
+  margin: 0 0 30px;
+}
+
+.h3CustomMobile {
+  visibility: hidden;
+}
+
+/* -------CSSFOR DIFFERENT SECTION-------- */
+.inerWrapperRoadMap {
+  display: flex;
+  justify-content: center;
+}
+
+.roadMApContainer {
```

Comment:
why you make same container here @PRABgit01  you can use same container for whole website not different for every section

Reply:
remove this container we need only one @PRABgit01